pkg/library/libraryinputresources/prune_mustgather.go

```diff
@@ -165,6 +204,20 @@ func GetRequiredInputResourcesForResourceList(ctx context.Context, resourceList
 	return instances.List(), errors.Join(errs...)
 }
 
+func getFilename(instance *unstructured.Unstructured, gvr schema.GroupVersionResource) string {
+	namespacedString := "REPLACE_ME"
+	if len(instance.GetNamespace()) > 0 {
+		namespacedString = "namespaces"
+	} else {
+		namespacedString = "cluster-scoped-resources"
+	}
+	group := gvr.Group
+	if len(group) == 0 {
+		group = "core"
+	}
+	return path.Join(namespacedString, instance.GetNamespace(), group, fmt.Sprintf("%s.yaml", gvr.Resource))
```

Comment:
this isn't accurate.  recall that the name can be a file with the name of the resource.yaml too.

Reply:
removed this altogether